lib/blpapi-wrapper.ts

```diff
@@ -324,7 +334,11 @@ export class Session extends events.EventEmitter {
                 throw new Error('Unknown request type ' + requestName);
             }
             var responseEventName = REQUEST_TO_RESPONSE_MAP[requestName];
-            this.session.request(uri, requestName, request, correlatorId);
+            if (isAuthRequest) {
```

Comment:
I am not a big fan of this code-reuse. I think I prefer having `request` & `authorizeUser` as separate implemented functions, cause I don't see lots of common behavior except the `listen` to event.